app/views/items/_embargo_form.html.erb

```diff
@@ -1,17 +1,14 @@
-<% if current_user && current_user.is_admin %>
```

Comment:
Hmm, this is removed and not replaced.  Assuming this now depends on the controller for authz.